README.md

```diff
@@ -213,6 +213,7 @@ lll: Reports long lines [fast: true, auto-fix: false]
 maligned: Tool to detect Go structs that would take less memory if their fields were sorted [fast: true, auto-fix: false]
 misspell: Finds commonly misspelled English words in comments [fast: true, auto-fix: true]
 nakedret: Finds naked returns in functions greater than a specified function length [fast: true, auto-fix: false]
+nolintlint: Reports ill-formed or insufficient nolint directives [fast: true, auto-fix: false]
```

Comment:
Can you make these changes to `README.tmpl.md` and regenerate the `README.md` from that?

Reply:
My understanding is that this is automatically generated already from somewhere else.  I just ran make README.md to create it, so I think this is right but please let me know if there is some other file I should be changing.

Reply:
You are right, @ashanbrown, it is expected for `README.md` to be re-generated after the `README.tmpl.md` change. The CI checks specifically for that, forbidding outdated `README.md`.

So it should be OK.